src/BulletChartColumns.ts

```diff
@@ -25,125 +25,105 @@
 import DataViewCategorical = powerbiVisualsApi.DataViewCategorical;
 import DataViewValueColumns = powerbiVisualsApi.DataViewValueColumns;
 import DataViewCategoryColumn = powerbiVisualsApi.DataViewCategoryColumn;
-import DataViewCategoricalColumn = powerbiVisualsApi.DataViewCategoricalColumn;
-import DataViewMetadataColumn = powerbiVisualsApi.DataViewMetadataColumn;
-import DataViewTable = powerbiVisualsApi.DataViewTable;
+import PrimitiveValue = powerbiVisualsApi.PrimitiveValue;
 
 import { converterHelper } from "powerbi-visuals-utils-dataviewutils";
 
-export class BulletChartColumns<T> {
-    public static GET_COLUMN_SOURCES(dataView: DataView) {
-        return this.GET_COLUMN_SOURCES_T<DataViewMetadataColumn>(dataView);
-    }
-
-    public static GET_TABLE_VALUES(dataView: DataView): BulletChartColumns<any[]> {
-        const table: DataViewTable = dataView && dataView.table,
-            columns = this.GET_COLUMN_SOURCES_T<any[]>(dataView);
-
-        return columns && table && lodashMapvalues(
-            columns, (n: DataViewMetadataColumn) => n && table.rows.map(row => row[n.index]));
-    }
-
-    public static GET_TABLE_ROWS(dataView: DataView): BulletChartColumns<any>[] {
-        const table: DataViewTable = dataView && dataView.table,
-            columns = this.GET_COLUMN_SOURCES_T<any[]>(dataView);
-
-        return columns && table && table.rows.map(row =>
-            lodashMapvalues(columns, (n: DataViewMetadataColumn) => n && row[n.index]));
-    }
-
-    public static GET_CATEGORICAL_VALUES(dataView: DataView): BulletChartColumns<any[]> {
-        const categorical: DataViewCategorical = dataView && dataView.categorical,
-            categories = categorical && categorical.categories || [],
-            values = categorical && categorical.values || <DataViewValueColumns>[],
-            series = categorical && values.source && this.GET_SERIES_VALUES(dataView);
-
-        return categorical && lodashMapvalues(new this<any[]>(), (n, i) =>
-            (<DataViewCategoricalColumn[]>Array.from(categories)).concat(Array.from(values))
-                .filter(x => x.source.roles && x.source.roles[i]).map(x => {
-                    const hasHighLight: boolean = !!(<DataViewValueColumn>x).highlights;
-                    let useHighlightAsValue: boolean;
-
-                    if (hasHighLight) {
-                        useHighlightAsValue = (<DataViewValueColumn>x).highlights.every(y => {
-                            if (y === null || y === undefined) {
-                                return false;
-                            }
-                            return true;
-                        });
-                    }
-
-                    return useHighlightAsValue ? (<DataViewValueColumn>x).highlights : (<DataViewValueColumn>x).values;
-                })[0]
-            || values.source && values.source.roles && values.source.roles[i] && series);
-    }
+const dataViewValueColumns = [
```

Comment:
I think, this variable naming is not good. It is no a value from the dataView. Also it is very similar to an interface import on line 32?

Reply:
Renamed to `bulletChartValueColumnsList`.